### ISSUE TITLE
Stripe API Libaray is missing or could not be loaded.

### ISSUE THREAD

**Issue by alanapplyby** [chronon/CakePHP-StripeComponent-Plugin#15]
Hi,

I am getting the error:

Stripe API Libaray is missing or could not be loaded.

I have loaded the plugin in bootstrap.php and added my keys.

I have:

app/Vendor/stripe/stripe-php

app/Plugin/Stripe

However it won't load.

Thanks,

Alan.

**Comment by chronon** [comment]
I highly suggest using composer, but if you can't for some reason the Stripe API library has to be in this plugin's Vendor directory, not `app/Vendor`. Check the README again for git installation instructions, it says this there.

**Comment by alanapplyby** [comment]
HI Gregory,

Thank you, I got it working now. I copied the file structure from a
previous install that I know works and for some reason it wouldn't work on
the new setup. However it now works at least.

I found a small typo, hope you don't mind me pointing it out. Line 73 of
stripe compoent, the word 'Library':

throw new CakeException('Stripe API Libaray is missing or could not be
loaded.');

Thank for the plugin- very useful.

Regards,

Alan.
